scripts/sct_resample.py

```diff
@@ -1,31 +1,24 @@
-#!/usr/bin/env python
 #########################################################################################
 #
 # Resample data.
 #
 # ---------------------------------------------------------------------------------------
 # Copyright (c) 2014 Polytechnique Montreal <www.neuro.polymtl.ca>
 # Authors: Julien Cohen-Adad, Sara Dupont
-# Modified: 2015-09-08
+# Modified: 2017-11-22 (moved the code to spinalcordtoolbox module)
```

Comment:
The change log is contained inside git history, does it make sense to keep these lines?

Reply:
no we can remove

Reply:
Removed.